pages/dkp/kaptain/2.1.0/install/air-gapped-2.1/index.md

```diff
@@ -79,7 +79,7 @@ If you need to run Spark jobs on Kubernetes using Spark Operator, you must insta
     - Download `kaptain-air-gapped-2.1.0.tar.gz` that will contain the required artifacts to perform an air-gapped installation.
     - (Optional) Download the custom image artifacts `kaptain-air-gapped-2.1.0_cpu.tar.gz` or `kaptain-air-gapped-2.1.0_gpu.tar.gz` based on whether you need CPU or GPU for your workloads.
 
-1.  Place the bundle in a location where you can load and push the images to your private Docker registry.
+1.  Unpack the bundle and the nested bundle within in a location where you can load and push the images to your private Docker registry.
```

Comment:
```suggestion
1.  Extract the bundle and the notices file and place the air gap images bundle in a location where you can load and push the images to your private Docker registry.
```

Reply:
Double nested archives are probably not something our customers will understand. :-)

Reply:
Just so we are consistent with other places
```suggestion
1.  Extract the bundle and the NOTICES.txt and place the air gap images bundle in a location where you can load and push the images to your private Docker registry.
```

Reply:
I am thinking that we should not directly refer to the notices.txt. Maybe? 

```suggestion
1.  Extract the image bundle archive and other files within a location where you can load and push the images to your private Docker registry.
```